.github/workflows/pipeline-segment-electron-test.yml

```
@@ -159,7 +165,39 @@ jobs:
           chown -R :builduser . && chmod -R g+w .
           chmod 4755 ../out/Default/chrome-sandbox
           runuser -u builduser -- git config --global --add safe.directory $(pwd)
-          runuser -u builduser -- xvfb-run script/actions/run-tests.sh script/yarn test --runners=main --trace-uncaught --enable-logging --files $tests_files
+          if [ "$IS_ASAN" == "true" ]; then
```

Let's use inputs where possible?
```suggestion
          if [ "${{ inputs.is-asan }}" == "true" ]; then
```